### PR TITLE
Xml

### PR DESCRIPTION
**Pull request by kasparvonbeelen**
update xml notebook

### CODE REVIEW

File: Sessions/3.2 - Reading XML Files.ipynb [hunk 1]
```diff
@@ -374,7 +507,6 @@
     "\n",
     "- Generally, attributes capture information that is \"descriptive of an element but not part of its content\" (e.g. identifiers.)\n",
     "- Attributes most be **quoted**\n",
```
Comment (mcollardanuy) [inline Sessions/3.2 - Reading XML Files.ipynb]: Typo: "must"

File: Sessions/3.2 - Reading XML Files.ipynb [hunk 2]
```diff
@@ -374,7 +507,6 @@
...
-    "- **Elements** structure what is **implicit**, the **attributes** often enrich this structure by adding information that is **absent**.\n",
     "- Attributes contained in the **start tag** (no attributes in the end-tag.)"
```
Comment (mcollardanuy) [inline Sessions/3.2 - Reading XML Files.ipynb]: "Attributes **are** contained"?

File: Sessions/3.2 - Reading XML Files.ipynb [hunk 3]
```diff
@@ -770,89 +1256,158 @@
...
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "# each element has different __Python__ attributes\n",
-    "root[1][1].tag"
+    "root[1][2].tag"
```
Comment (mcollardanuy) [inline Sessions/3.2 - Reading XML Files.ipynb]: I got an `IndexError: list index out of range` error.

**Review by mcollardanuy — COMMENTED** [review]
Hi @kasparvonbeelen! All good! I've left a couple of comments in the code. Other than that, I'd suggest wrapping xml examples in markdown xml code blocks (as you do in 1.1 Toy examples), because otherwise most of them look like plain text.

 Other than that, I'd clear all outputs before merging.

File: Sessions/3.2 - Reading XML Files.ipynb (hunk 1 above)
Comment (kasparvonbeelen) [reply Sessions/3.2 - Reading XML Files.ipynb]: we sputted ;-)

File: Sessions/3.2 - Reading XML Files.ipynb (hunk 2 above)
Comment (kasparvonbeelen) [reply Sessions/3.2 - Reading XML Files.ipynb]: Changed to: "Attributes are embedded in the start tag"

File: Sessions/3.2 - Reading XML Files.ipynb (hunk 3 above)
Comment (kasparvonbeelen) [reply Sessions/3.2 - Reading XML Files.ipynb]: I removed `root[1][2].tag`

**Review by mcollardanuy — APPROVED** [review]
All good!

**Pull request merged**